modules/apps/headless/headless-builder/headless-builder-test/src/testFunctional/tests/[APIRest]APIGenerator/ExposeQueryParametersInAPIExplorerForGeneratedAPI.testcase

```diff
@@ -0,0 +1,548 @@
+@component-name = "portal-headless"
+definition {
+
+	property custom.properties = "feature.flag.LPS-167253=true${line.separator}feature.flag.LPS-178642=true";
+	property portal.release = "true";
+	property portal.upstream = "true";
+	property testray.component.names = "Object";
+	property testray.main.component.name = "Headless Builder";
+
+	setUp {
+		TestCase.setUpPortalInstance();
+
+		User.firstLoginUI();
+	}
+
+	tearDown {
+		ApplicationAPI.deleteAllAPIApplication();
+
+		JSONObject.deleteAllCustomObjects();
+
+		var testLiferayVirtualInstance = PropsUtil.get("test.liferay.virtual.instance");
+
+		if (${testLiferayVirtualInstance} == "true") {
+			PortalInstances.tearDownCP();
+		}
+	}
+
+	@priority = 5
+	test CanApplyFilterParameter {
+		property portal.acceptance = "true";
+
+		task ("Given custom object Student with name text field is created and published") {
```

Comment:
Can we put create Student object definition and Student entries(Able, Tom and Test) into Setup()?

Reply:
but that's not all the case need this,we still add this in setup?

Reply:
@magjed4289 what do you think of it?

Reply:
@magjed4289 I have moved the Student object definition and Student entries in Setup()